app/javascript/react/components/SuperpowerDetailTile.js

```diff
@@ -8,10 +8,6 @@ const SuperpowerDetailTile = (props) => {
         <img src="#" alt="Picture of the selected superpower" />image tag
       </div>
```

Comment:
This one doesn't look to be quite complete either, as it doesn't have an image but just the text - we'd want to finish this up before merging this component to master as well! We want each and every PR to be a complete feature.